types/address/hash_test.go

```diff
@@ -0,0 +1,90 @@
+package address
```

Comment:
could we use table tests in this file?

Reply:
I was thinking about it. I don't think it will bring any benefit here. We are doing some setups, different preparation .... it would only obfuscate it here.